app/elements/candidates/invite-candidate-input.html

```diff
@@ -54,28 +45,44 @@
 				challenges: {
 					type: Array
 				},
-				challenge: {
-					type: Object
-				},
 				candidate: {
 					type: Object,
 					reflectToAttribute: true,
 					notify: true
-				},
-				singleChallenge: {
-					type: Boolean,
-					computed: 'isSingleChallenge(challenge)'
 				}
 			},
-			isSingleChallenge: function(challenge){
-				return challenge !== undefined;
-			},
-			_setChallenge: function(){
+			observers: [
+				'_afterProperties(challenges, candidate)'
+			],
+			_setChallenge: function () {
 				this.candidate.Challenge = this.$.challenges.options[this.$.challenges.selectedIndex].value;
 			},
-			_removeItem: function(){
+			_removeItem: function () {
 				this.candidate.Removed = true;
 				this.remove();
+			},
+			_afterProperties: function () {
+				var option = document.createElement('option');
+				if (this.challenges.length === 1) {
+					option.value = this.challenges[0].Key;
+					option.innerHTML = this.challenges[0].Name;
```

Comment:
This looks like shit and inconsistent with when there is a select on the modal. 
so i will waste more time doing css so it looks decent 1 day before the presentation.

![screenshot from 2015-09-08 16-25-30](https://cloud.githubusercontent.com/assets/9076897/9735844/45d18d14-5646-11e5-871f-86fcf0735c83.png)